config/controller.yaml.example

```diff
@@ -21,4 +22,12 @@ embed_etcd:
   election-timeout: 600000
   clusters:
     - test-1=http://127.0.0.1:2380
-secret_encryption_salt: "encryption_salt"
```

Comment:
why delete it

Reply:
I didn't see this, what's the parameter for?

Reply:
recover it